sktime/forecasting/ltsf.py

```diff
@@ -0,0 +1,74 @@
+"""Deep Learning Forecasters using LTSF-Linear Models."""
+
+import torch
+
+from sktime.forecasting.deep_learning.base import BaseDeepNetworkPyTorch
```

Comment:
Do we want the BaseModel under forecasting or under network, where the existing CCN are?

Reply:
I was using the `sktime` classifiers as an example, their `BaseDeepClassifier` is found in `sktime/classification/deep_learning/base.py`